.github/workflows/tests.yml

```
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ['3.2', '3.1', '3.0']
+        ruby-version: ['3.4', '3.3']
```

@JuanVqz I think for now we want to keep testing 3.0, 3.1, 3.2

---

I just thought that since we are already running Ruby 3.3 and 3.4 for our projects it was fine to remove support for it in the next release, but happy to know what do you think

---

@JuanVqz I'm not sure *all of our projects* are using Ruby 3.3 and up. For example: https://github.com/fastruby/librarian/blob/main/Gemfile 

Could you please make sure this project still works with Ruby 3.1? 

```suggestion
        ruby-version: ['3.4', '3.3', '3.2', '3.1']
```

---

@etagwerker done! thanks for reviewing it